src/ucs/debug/log_def.h

```diff
@@ -84,6 +84,16 @@ BEGIN_C_DECLS
     } while(0)
 
 
+#define ucs_log_dump_hex_buf(_data, _length) \
+    do { \
+        if (ucs_global_opts.log_print_queues) { \
```

Comment:
Consider refactoring a bit. The name of this macro suggests it is a "generic" way to dump hex buffer into log, not particularly coupled to "queues". Then this `if` shouldn't be in it.